websites/H/Hyakanime/presence.ts

```diff
@@ -0,0 +1,56 @@
+const presence = new Presence({
+		clientId: "967174885992456292"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "applogo_7fac0ec4359bda8ccf0f",
+			startTimestamp: browsingTimestamp
+		},
+		page = document.location.pathname;
+	if (page.startsWith("/anime/")) {
+		presenceData.details = "Regarde la fiche d'un animé:";
+		presenceData.state = document.querySelector(
+			"#root > div.anime-body > div.anime-white-informations > div.anime-white-infos > div:nth-child(2) > h1"
+		).textContent;
+		presenceData.buttons = [
+			{
+				label: "Fiche",
+				url: document.location.href
+			}
+		];
+	} else if (page.startsWith("/profil/")) {
+		presenceData.details = "Regarde le profil de:";
+		presenceData.state = document.querySelector(
+			"#root > div:nth-child(3) > div.profil-banner-top > div > div.bloc-name-banner > h1"
+		).textContent;
+		presenceData.buttons = [
+			{
+				label: "Profil",
```

Comment:
Button labels should have a verb in the infinitive e.g. "View profile" (translate accordingly)